### DISCUSSION TITLE
Possible to have separate shortcuts for opening new tab as independent tab and sibling/child?

### DISCUSSION THREAD

**Discussion by fragande** [piroor/treestyletab#2862] [General]
I'm not sure if this is already possible or not. I want to be able to open a new tab as either an independent tab _or_ sibling/child of current tab depending on the scenario using two separate shortcuts. My current "solution" is to have the usual Ctrl+T or double click on empty space opening the tab as sibling to current tab and Ctrl+LB on the New Tab button set as new independent tab. This is however very suboptimal as one have to move the cursor to the New Tab button every time.

Ideally I would like to be able to have two separate keyboard shortcuts, one for each option.

**Comment by piroor** [comment]
Did you try Firefox's option UI to customize addon shortcuts?
https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox

**Reply by fragande** [reply]
Wow, no, I was not aware of that feature at all actually. That's exactly what I was looking for, thank you very much!
